buildSrc/src/main/kotlin/eternalcore-repositories.gradle.kts

```diff
@@ -5,11 +5,8 @@ plugins {
 repositories {
     mavenCentral()
     maven { url = uri("https://hub.spigotmc.org/nexus/content/repositories/snapshots/") }
-    maven { url = uri("https://repo.panda-lang.org/releases/") }
     maven { url = uri("https://repo.papermc.io/repository/maven-public/") }
-    maven { url = uri("https://repository.minecodes.pl/releases/") }
-    maven { url = uri("https://repository.minecodes.pl/snapshots/") }
-    maven { url = uri("https://repo.eternalcode.pl/snapshots/") }
+    maven { url = uri("https://repo.panda-lang.org/releases/") }
```

Comment:
repositories could be sorted alphabetically

Reply:
Alphabetically sorting doesn't make sense (also for dependencies or other things like as fields). Currently we sort repositories by frequency of use (by dependencies) to reduce dependency resolving time.

Reply:
Yessirski